websites/R/Raven Code/presence.ts

```diff
@@ -0,0 +1,98 @@
+const presence = new Presence({ clientId: "937015924425367643" });
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "logo",
+			details: "Geziniyor.",
```

Comment:
I'd say remove the dot at the end of this and the next ones. It's not a complete sentence nor something needed.